OrbitGl/FrameTrack.cpp

```diff
@@ -18,20 +18,32 @@ constexpr const uint64_t kHeightCapAverageMultipleUint64 = 6;
 constexpr const float kBoxHeightMultiplier = 3.f;
 }  // namespace
 
+float FrameTrack::GetMaximumScaleFactor() const {
+  float scale_factor = 0.f;
+  if (stats_.average_time_ns() > 0) {
+    scale_factor =
+      static_cast<double>(stats_.max_ns()) / static_cast<double>(stats_.average_time_ns());
+    scale_factor = std::min(scale_factor, static_cast<float>(kHeightCapAverageMultipleDouble));
+  }
+  return scale_factor;
```

Comment:
More of a nitpick for me, but others pointed this out in PRs a lot - structuring like so could be easier to follow:
```
if (stats_.average_time_ns() == 0) {
  return 0.f;
}

return static_cast<float>(stats_.max_ns()) / stats_.average_time_ns();

Reply:
I think that's a good idea, done.